packages/cli/src/lib/utils.test.ts

```diff
@@ -71,6 +72,24 @@ describe('utils', () => {
   });
 
   describe('getAxeSource', () => {
+    describe('mock file', () => {
+      beforeEach(() => {
+        mock({
+          '/node_modules/axe-core': {},
+          '../node_modules/axe-core': {
+            'axe.js': 'foobar'
+          }
+        });
+      });
+
+      afterEach(() => {
+        mock.restore();
+      });
+      it('fall back to use `locally` installed axe-core', () => {
+        const axeSource = utils.getAxeSource();
+        assert.isNotNull(axeSource);
```

Comment:
I'm not sure what this is supposed to prove, but just asserting that it isn't null doesn't mean much.